main.py

```diff
@@ -0,0 +1,123 @@
+import discord, datetime, time, random # Imports discord.py, datetime, time and random libraries
+from discord.ext import commands # Import the extension for discord.py - discord.ext command
+from discord import Member # import discord member, used in warning
+import json # import the json library
+import os
+
+from math import sqrt, cos, sin # import more math functions
+
+prefix = "m/" # Set the prefix. e.g "!sb "
+bot = commands.Bot(command_prefix=prefix) # Define what bot is
+bot.remove_command('help') # Remove the default help command from the Discord.py commands lib.
+botver = "1.0 [beta]" # Set the bot version number.
+
+start_time = time.time() # Starts the timer for the uptime of the bot.
+
+functions = ['+', '-', '*', '/', 'sqrt', 'cos', 'sin'] # math functions
+
+# create bot memory structure
+if not os.path.isdir("./.botmem"): os.mkdir("./.botmem") 
+if not os.path.exists("./.botmem/warnings.json"): open("./.botmem/warnings.json", "w+")
+if open("./.botmem/warnings.json", "r").read() == "":
+    warnings = {}
+else:
+    warnings = json.loads(open("./.botmem/warnings.json").read())
+##############################
+
+@bot.event # When the bot first loads up
+async def on_ready():
+    print(""" _______
+||  _  ||
+|| | | || version {} """.format(botver))
+    print('The bot has logged in as {0.user}'.format(bot)) # Say that [bot name] is logged on in the terminal
+    await bot.change_presence(activity=discord.Game(name=f"m/help - {botver}")) # Change the bot activity
+
+@bot.command() # Help command. This will give you all of the commands
+async def help(ctx):
+    await ctx.send("""Minimal | By Cob:web Development 
+m/help                   - Shows this message
+m/about                  - Shows the bot statistics and ping
+m/calculate              - Calculate basic math. +-/*sqrt()sin()cos()
+m/warn <memberMention>   - Warn a member
+m/unwarn <memberMention> - Unwarn a member
+m/status <memberMention> - get warning status of member""")

```

Comment:
Shouldn't we use arguments so that we can have the command `m/warn remove <mention>` rather than different commands, we can use a command handler for that feature.